build.gradle

```diff
@@ -37,7 +36,7 @@ if (cliTasks.contains("rc")) {
 
 dependencies {
     implementation 'org.codehaus.groovy:groovy-all:2.4.15'
-    api 'org.kohsuke:github-api:1.131'
+    api 'org.kohsuke:github-api:[1.95,2)'
```

Comment:
Was there issues with the 1.131? The newer versions from this library were associated with less problems coming from failing github requests.

Reply:
hmm must be an artict from rebasing it. But `[1.95,2)` means bigger 1.95 and smaller 2.0